advanced/Scripts/chronometer.sh

```diff
@@ -177,7 +224,7 @@ get_sys_stats() {
       ph_lte_ver="${ph_ver_raw[1]}"
       ph_ftl_ver="${ph_ver_raw[2]}"
     else
-      ph_core_ver="${COL_LIGHT_RED}API unavailable${COL_NC}"
+      ph_core_ver="${COL_LIGHT_RED}Offline${COL_NC}"
```

Comment:
If the version can't be obtained, then I make the assumption that the `pihole` command is broken, or not installed. "Offline" is just wording that fits on a smaller screen, unlike "API unavailable" which is technically more accurate.